File: C++/Harry and big doughnuts.cpp
#include <iostream>

int main() 
{
    int t, c, k, w;;
    std::cin >> t;
    while(t--) 
    {
        std::cin >> c >> k >> w;
        if (c * w <= k)
            std::cout << "yes\n";
        else
            std::cout << "no\n";
    }
    return 0;
}

File: Java/Harry and big doughnuts.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();
        for(int i = 0; i < t; i++)
        {
            int c = input.nextInt();
            int k = input.nextInt();
            int w = input.nextInt();
            if (c * w <= k)
                System.out.println("yes");
            else
                System.out.println("no");
        }
        input.close();
    }
}
